extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -87,7 +87,17 @@ class AmpSocialShare extends AMP.BaseElement {
 
     const hrefWithVars = addParamsToUrl(this.shareEndpoint_, this.params_);
     const urlReplacements = Services.urlReplacementsForDoc(this.getAmpDoc());
-    urlReplacements.expandAsync(hrefWithVars).then(href => {
+    const bindingVars = typeConfig['bindings'];
+    let bindings;
+    if (bindingVars) {
+      bindings = {};
```

Comment:
Nit: We could probably move this to the `let bindings` above (making it a `const`).